### DISCUSSION TITLE
Mac deprecations

### DISCUSSION THREAD

**Discussion by TWiStErRob** [detekt/detekt#5471] [Q&A] [answered]
hey @3flex @schalkms, I noticed a Mac deprecation on Detekt:
https://github.com/detekt/detekt/actions/runs/3309796152
> Pre Merge Checks: .github#L1
> macOS-latest pipelines will use macOS-12 soon. For more details, see https://github.com/actions/runner-images/issues/6384

I blamed and notice you were involved in adding Mac support in [Travis](https://github.com/detekt/detekt/blame/1.6.0/.travis.yml).
I'm on Windows + don't know Detekt CI enough, would you mind scanning the link they provided to see if there's anything needs to be done?

**Comment by schalkms** [comment, marked as answer]
Thanks for the hint!
I don't think that this upgrade of MacOS will raise any problems. The ruby versions might affect the website generator, but this GitHub action runs on `ubuntu-latest`.

**Reply by schalkms** [reply]
Nevertheless, I created a PR for testing purposes. Let's see whether it will run or not.

**Reply by TWiStErRob** [reply]
Haha, good point, easy testing :)
